frontend/src/tabs/NotificationPanel.css

```diff
@@ -0,0 +1,39 @@
+.notification-panel {
+    position: fixed;
+    right: -100%; 
+    top: 20px;
+    width: 300px;
+    padding: 20px;
+    background-color: #444;
+    color: #fff;
+    border-radius: 8px;
+    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+    transition: right 0.5s ease-in-out;
+    z-index: 1000;
+  }
+  
+  .notification-panel.slide-in {
+    right: 20px;
+  }
+  
+  .notification-panel.slide-out {
+    right: -100%;
+  }
+  
+  .notification-content {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: #000000E6;
+  }
+  
+  .notification-content button {
```

Comment:
There are styles here for close buttons, to make them match the style of the other panels : 
https://github.com/keep-starknet-strange/art-peace/blob/b7122c3eedf5e2be34ea392c189e8b15d4d2f84e/frontend/src/tabs/BasicTab.css#L27
https://github.com/keep-starknet-strange/art-peace/blob/b7122c3eedf5e2be34ea392c189e8b15d4d2f84e/frontend/src/utils/Styles.css#L99